src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -1231,20 +1231,22 @@ void Creature::SelectLevel()
 {
     CreatureTemplate const* cInfo = GetCreatureTemplate();
 
-    if (!HasScalableLevels())
-    {
-        // level
-        uint8 minlevel = std::min(cInfo->maxlevel, cInfo->minlevel);
-        uint8 maxlevel = std::max(cInfo->maxlevel, cInfo->minlevel);
-        uint8 level = minlevel == maxlevel ? minlevel : urand(minlevel, maxlevel);
-        SetLevel(level);
-    }
-    else
+    // level
+    uint8 minlevel = std::min(cInfo->maxlevel, cInfo->minlevel);
+    uint8 maxlevel = std::max(cInfo->maxlevel, cInfo->minlevel);
+    uint8 level = minlevel == maxlevel ? minlevel : urand(minlevel, maxlevel);
+    SetLevel(level);
```

Comment:
Sniffes confirmed, levels are allways set.
Did not check legion creatures yet

Reply:
I don't know how it is in 735 but.. in 7.2 and lower.. if the creature got a scaling level, his NORMAL level was always MAX.. which you removed now

Reply:
Normal Level in sniffs is Standard randomized min/max Level

Reply:
Now that's a not totally useless and duplicate data database scheme

Reply:
This is wrong, levels are not random between min & max, all outland creature UNIT_FIELD_LEVEL (so scaling 64-80, 1690 query) where between 64 & 68, doesn't seem random to me

Reply:
Traesh.. 
UNIT_FIELD_LEVEL is untouched by blizzard. This Field is Set like no scaling exist and that is What i say since this PR is opened.
We are talking about normal Level and not scaling.
In the actual scaling System Level is similar to scalingmax and that would be wrong. That's why i Set it that Level is Set Normaly.

Reply:
Sure, perfectly understand why you change it and you're right when you say it's not equal to maxlevel anymore

I'm just saying that in the full outland sniff i have, all UNIT_FIELD_LEVEL are between 64 and 68, so they are not totally random, because if they were indeed random they would statistically be between 64 and 80

The other option is that Blizzard stopped setting UNIT_FIELD_LEVEL to max scaling level, and updated all standard creature_template min&max level to 64-68, for no reason at all. Which would be very weird (but still possible when it come to Blizzard)

Reply:
I dont think they updated ay template levels.. 
Couldnt find anything in whole azeroth

Reply:
> Blizzard stopped setting UNIT_FIELD_LEVEL to max scaling level

They never did, it just seemed to be like that, because Legion Creatures did never have any minmaxlevel because they were never unscaled.

Reply:
> and updated all standard creature_template min&max level to 64-68

Can be possible, but unsure.
They did not touch any levels at Kalimdor or Eastern Kingdoms.